### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by Joshcarp**
- Adds More documentation, as-well as fixes old documentation
- Replaces png with svg files
@anz-bank/sysl-developers

### CODE REVIEW

File: cmd/sysl/tests/data.sysl [hunk 1]
```diff
@@ -6,6 +6,7 @@ Project [seqtitle="DataModel"]:
 
 RelModel:
     !table Employee:
+        # ~ Are patterns HTTP body post would be [~body] or something; giving more information
```
Comment (anzdaddy) [inline cmd/sysl/tests/data.sysl]: This comment isn't super clear.

File: demo/examples/Data-Model-Diagrams/1_project.sysl [hunk 2]
```diff
@@ -1,15 +1,25 @@
...
+        id <: Server.id
 
-    !type Customer:
-        customer_id <: int
-        orders <: set of Order
+AnotherOne:
+    !type name:
+        firstName <: string
+        lastName <: string
+
```
Comment (anzdaddy) [inline demo/examples/Data-Model-Diagrams/1_project.sysl]: Weird EOF behaviour.

File: demo/examples/Control-Flow/3_project.svg [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+PlantUML version 1.2019.13(Tue Dec 10 17:18:29 UTC 2019)
+(GPL source distribution)
+Java Runtime: Java(TM) SE Runtime Environment
+JVM: Java HotSpot(TM) 64-Bit Server VM
+Java Version: 1.7.0_25-b15
+Operating System: Linux
+Default Encoding: UTF-8
+Language: en
+Country: US
+--></g></svg>
```
Comment (anzdaddy) [inline demo/examples/Control-Flow/3_project.svg]: Missing \n

Comment (Joshcarp) [reply demo/examples/Control-Flow/3_project.svg]: Wait this is an auto generated file?

Comment (Joshcarp) [reply demo/examples/Control-Flow/3_project.svg]: Yeah, I've got a feeling this should be a bug report or something

**Pull request merged**